h2o-core/src/main/java/hex/Model.java

```diff
@@ -422,6 +422,14 @@ public long getOrMakeRealSeed(){
      */
     public double _max_runtime_secs = 0;
 
+    /** Using _main_model_time_budget_factor to determine if and how we should restrict the time for the main model.
+     * In general, we should use 0 or > 1 to be reasonably certain that the main model will have time to converge.
+     * if _main_model_time_budget_factor < 0 use max(1, -_main_model_time_budget_factor * remaining time) as max runtime secs
+     * if _main_model_time_budget_factor == 0 do not restrict time for the main model
+     * if _main_model_time_budget_factor > 0 use max(remaining time, _main_model_time_budget_factor * time for a single cv model) as max runtime secs
+     */
```

Comment:
same as below: you shouldn't have to replicate the logic defined in `setMaxRuntimeSecs` here. 
You could just explain shortly and refer to the details using `{@link ModelBuilder#setMaxRuntimeSecsForMainModel()}